Sources/Classes/RSPreferenceManager.m

```diff
@@ -255,4 +256,27 @@ - (BOOL) getAutoTrackingStatus {
     return [[NSUserDefaults standardUserDefaults] boolForKey:RSSessionAutoTrackStatus];
 }
 
+- (void) saveEventDeletionCompletedStatus {
```

Comment:
Can we rename it to `saveEventDeletionCompleted` (or) `saveEventDeletionStatus` ?

Reply:
Yeah, I'll rename it to `saveEventDeletionStatus`.